DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/_DelegateCourseInfoCard.cshtml

```diff
@@ -60,7 +60,7 @@
           <partial name="_SummaryFieldValue" model="@Model.Completed" />
           <dd class="nhsuk-summary-list__actions">
             @if (Model.Completed == null) {
-              <a href="#">Set completed date</a>
+              <a href="/TrackingSystem/Delegates/@DelegateProgressAccessRoute.ViewDelegate/DelegateProgress/@Model.ProgressId/EditCompletionDate">Set completed date</a>

```

Comment:
Same here. Is there a reason we aren't using the asp-controller, asp-action etc for this?